charts/kueue/values.yaml

```diff
@@ -151,11 +151,16 @@ webhookService:
       targetPort: 9443
   type: ClusterIP
 
-# kueue-viz dashboard
-enableKueueViz: false
```

Comment:
No need for moving the section, let's minimize the diff in case we need to cherry-pick it.

Reply:
> No need for moving the section, let's minimize the diff in case we need to cherry-pick it.

+1

Reply:
done.